pytorch_lightning/core/hooks.py

```diff
@@ -138,11 +138,20 @@ def backward(self, use_amp, loss, optimizer):
                 else:
                     loss.backward()
 
+        .. note:: with PyTorch 1.6+ + precision=16 + multiple optimizers, set .backward(retrain_graph=True)
+
         """
         if trainer.precision == 16:
-
             # .backward is not special on 16-bit with TPUs
-            if not trainer.on_tpu:
+            if trainer.on_tpu:
+                return
+
+            if self.trainer.use_native_amp:
+                # don't forget to retain graph on backward with multiple optimizers
```

Comment:
also remove this comment, see https://github.com/PyTorchLightning/pytorch-lightning/pull/1561/files#r413230111.